docs/azure/architecture.md

```diff
@@ -0,0 +1,71 @@
+---
+nav_order: 1
+parent: Kernel Memory in Azure
+title: Architecture
+permalink: /azure/architecture
+layout: default
+---
+
+# Azure Infrastacture Architecture
+
+Infrastructure of Kernel Memory in Azure is explained in this section, implementing an asynchronous microservices architecture.
```

Comment:
"This section explains the infrastructure of Kernel Memory in Azure, which implements an asynchronous microservices architecture."

Better flow